Firebase Push Notifications.md

```diff
@@ -0,0 +1,199 @@
+# Firebase push notifications
+
+We used to send push notifications directly to APNS or GCM from our code. We were maintaining device_ids, OS types, GCM/APNS certificates and sometimes this was really tricky.
```

Comment:
~~... and sometimes this was really tricky.~~
`. This approach was complicated and hard to maintain.`

Zelim ovdje naglasiti da zasto smo odustali od tog principa